ui/src/app/ui-banner/ui-banner.tsx

```diff
@@ -29,10 +30,11 @@ export const Banner = (props: React.Props<any>) => {
                     }
                 }
                 show = show && visible;
```

Comment:
Small bug: permanent banner is not shown if the user hides it before it was permanent. This change should fix it:
```suggestion
                show = permanent || show && visible;
```

Reply:
Thanks @alexmt for the review.  This line has been updated.